hackathon_site/hardware/models.py

```diff
@@ -95,3 +96,22 @@ class Incident(models.Model):
 
     def __str__(self):
         return self.id
+
+
+class AnnotatedHardwareManager(models.Manager):
+    def get_queryset(self):
+        return super()
```

Comment:
This whole thing needs to be wrapped in brackets, not valid python syntax as it is